src/Calculations/Mode.py

```diff
@@ -0,0 +1,21 @@
+from src.Calculations import Calculation
+import numpy
+import stats
+
+class Mode(Calculation.Calculation):
```

Comment:
how does this calculation behave where there are more than one mode in a rc? example: `[1, 1, 3, 4, 4]`

Reply:
It returns the first mode